cloudwatch-alarms-cloudfront.tf

```diff
@@ -0,0 +1,87 @@
+variable "default_threshold" {
```

Comment:
Keep variables in the _variables.tf files.

Reply:
I would leave those variables on the "resource" file because I don't want them to be assigned directly as module parameters. It should be passed as a map because it allows more flexibility when managing more than one alarm.